pkg/generators/forwarding/fluentd/fluent_conf_test.go

```diff
@@ -1088,7 +1186,56 @@ var _ = Describe("Generating fluentd config", func() {
                 time_format %Y-%m-%dT%H:%M:%S.%N%z
               </parse>
             </source>
-
+			<label @MEASURE>
```

Comment:
Indention seems off

Reply:
updated... but this particular test has some formatting issues